[DO NOT MERGE] EUI Visual Refresh testing branch

mgadewoll opened the pull request:
## Summary

>[!IMPORTANT]
This is a testing PR only that's a direct copy of https://github.com/elastic/kibana/pull/224382

mgadewoll committed:
refactor: adds colorModes prop to handle global search color modes

mgadewoll committed:
refactor: update JSON import path

mgadewoll committed:
fix: update token to ensure filter button group border is rendered

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
Cloud deployment initiated, see credentials at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/274

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/317312)
* Commit: acc5d9ebc6c86bc3b7d00bbb3f801b24a2fbd47b
* [Cloud Deployment](https://kibana-pr-227054.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #316859](https://buildkite.com/elastic/kibana-pull-request/builds/316859) failed ebe3d9bb2e9d892dc30c33842781d276008fd618

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317312","number":317312,"commit":"acc5d9ebc6c86bc3b7d00bbb3f801b24a2fbd47b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316859","number":316859,"commit":"ebe3d9bb2e9d892dc30c33842781d276008fd618"}],"number":317312}
buildkite-pr-comment-->

mgadewoll committed:
chore: update eui packages

mgadewoll committed:
refactor: update inTableSearchControl styles

mgadewoll committed:
test: update tests and snapshots

- color updates are triggered by updated initial option for EuiColorVisStore to ensure Borealis values

mgadewoll committed:
chore: update eui packages

mgadewoll committed:
refactor: cleanup force click

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/320550)
* Commit: bfef83fdd860fff4b94b2191b7f79891f1a620b4

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/320550#01981909-e824-4ee0-bfcb-054080154a57)

### History
* :broken_heart: [Build #320458](https://buildkite.com/elastic/kibana-pull-request/builds/320458) failed 8c0ec7c35f1b893c6fabca279401442853b31f8c
* :green_heart: [Build #320321](https://buildkite.com/elastic/kibana-pull-request/builds/320321) succeeded ce11739b7264e98f2f48be8509ab38bc7918a609
* :broken_heart: [Build #320084](https://buildkite.com/elastic/kibana-pull-request/builds/320084) failed ce11739b7264e98f2f48be8509ab38bc7918a609
* :broken_heart: [Build #319814](https://buildkite.com/elastic/kibana-pull-request/builds/319814) failed 3b549a536399afa3dae49574f205e30b61711093
* :green_heart: [Build #317312](https://buildkite.com/elastic/kibana-pull-request/builds/317312) succeeded acc5d9ebc6c86bc3b7d00bbb3f801b24a2fbd47b

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320550","number":320550,"commit":"bfef83fdd860fff4b94b2191b7f79891f1a620b4"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320458","number":320458,"commit":"8c0ec7c35f1b893c6fabca279401442853b31f8c"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320321","number":320321,"commit":"ce11739b7264e98f2f48be8509ab38bc7918a609"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320084","number":320084,"commit":"ce11739b7264e98f2f48be8509ab38bc7918a609"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/319814","number":319814,"commit":"3b549a536399afa3dae49574f205e30b61711093"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317312","number":317312,"commit":"acc5d9ebc6c86bc3b7d00bbb3f801b24a2fbd47b"}],"number":320550}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/320572)
* Commit: bfef83fdd860fff4b94b2191b7f79891f1a620b4

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/320572#01981941-055b-48cb-ba67-237fe1363c6e)

### History
* :broken_heart: [Build #320557](https://buildkite.com/elastic/kibana-pull-request/builds/320557) failed 3f32d694beaeb7ab69aa697ce9852f180c7a4048
* :broken_heart: [Build #320550](https://buildkite.com/elastic/kibana-pull-request/builds/320550) failed bfef83fdd860fff4b94b2191b7f79891f1a620b4
* :broken_heart: [Build #320458](https://buildkite.com/elastic/kibana-pull-request/builds/320458) failed 8c0ec7c35f1b893c6fabca279401442853b31f8c
* :green_heart: [Build #320321](https://buildkite.com/elastic/kibana-pull-request/builds/320321) succeeded ce11739b7264e98f2f48be8509ab38bc7918a609
* :broken_heart: [Build #320084](https://buildkite.com/elastic/kibana-pull-request/builds/320084) failed ce11739b7264e98f2f48be8509ab38bc7918a609

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320572","number":320572,"commit":"bfef83fdd860fff4b94b2191b7f79891f1a620b4"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320557","number":320557,"commit":"3f32d694beaeb7ab69aa697ce9852f180c7a4048"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320550","number":320550,"commit":"bfef83fdd860fff4b94b2191b7f79891f1a620b4"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320458","number":320458,"commit":"8c0ec7c35f1b893c6fabca279401442853b31f8c"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320321","number":320321,"commit":"ce11739b7264e98f2f48be8509ab38bc7918a609"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320084","number":320084,"commit":"ce11739b7264e98f2f48be8509ab38bc7918a609"}],"number":320572}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/320578)
* Commit: 0f4e9a95516dd315c43afc75920771282652fd68
* [Cloud Deployment](https://kibana-pr-227054.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #320572](https://buildkite.com/elastic/kibana-pull-request/builds/320572) failed bfef83fdd860fff4b94b2191b7f79891f1a620b4
* :broken_heart: [Build #320557](https://buildkite.com/elastic/kibana-pull-request/builds/320557) failed 3f32d694beaeb7ab69aa697ce9852f180c7a4048
* :broken_heart: [Build #320550](https://buildkite.com/elastic/kibana-pull-request/builds/320550) failed bfef83fdd860fff4b94b2191b7f79891f1a620b4
* :broken_heart: [Build #320458](https://buildkite.com/elastic/kibana-pull-request/builds/320458) failed 8c0ec7c35f1b893c6fabca279401442853b31f8c
* :green_heart: [Build #320321](https://buildkite.com/elastic/kibana-pull-request/builds/320321) succeeded ce11739b7264e98f2f48be8509ab38bc7918a609
* :broken_heart: [Build #320084](https://buildkite.com/elastic/kibana-pull-request/builds/320084) failed ce11739b7264e98f2f48be8509ab38bc7918a609

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320578","number":320578,"commit":"0f4e9a95516dd315c43afc75920771282652fd68"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320572","number":320572,"commit":"bfef83fdd860fff4b94b2191b7f79891f1a620b4"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320557","number":320557,"commit":"3f32d694beaeb7ab69aa697ce9852f180c7a4048"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320550","number":320550,"commit":"bfef83fdd860fff4b94b2191b7f79891f1a620b4"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320458","number":320458,"commit":"8c0ec7c35f1b893c6fabca279401442853b31f8c"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320321","number":320321,"commit":"ce11739b7264e98f2f48be8509ab38bc7918a609"}],"number":320578}
buildkite-pr-comment-->

mgadewoll committed:
chore: update i18n
